### DISCUSSION TITLE
View count in GA vs Vercel Audiences

### DISCUSSION THREAD

**Discussion by jaseflow** [vercel/community#2093] [Help] [answered]
### Summary

I see a big difference in users and view count in Google Analytics. 
## Google Analytics Last 24 hours
![image](https://user-images.githubusercontent.com/1094501/232305659-bcf2e7fd-49f3-4ce2-852f-6b77cff44869.png)

## Audiences Last 24 hours
![image](https://user-images.githubusercontent.com/1094501/232305640-5b159b11-793d-4f33-80e6-a616093612cb.png)

I have implemented the recommended way of adding `<Analytics />` to the bottom of `_app.tsx`.

Any ideas why this is?

**Comment by leerob** [comment, marked as answer]
Vercel Web Analytics is a privacy-friendly, first-party analytics tool, running securely on the server. With tools like Google Analytics, many Chrome Extensions will automatically block requests for this third-party analytics tool. The difference you're seeing here likely reflects those blocked users of Google Analytics.

**Comment by jaseflow** [comment]
I thought that might be the case. If that is so, that's good news for me :)
